bindings/python/crocoddyl/multibody/frames.cpp

```diff
@@ -198,6 +207,15 @@ void exposeFrames() {
                             deprecated<bp::return_value_policy<bp::return_by_value> >("Deprecated. Use id.")),
           bp::make_function(&FrameCoPSupport::set_id, deprecated<>("Deprecated. Use id.")), "frame ID")
       .def(PrintableVisitor<FrameCoPSupport>());
+
+  pinocchio::python::StdAlignedVectorPythonVisitor<FramePlacement, true>::expose("StdVec_FramePlacement");
```

Comment:
Can we use our implementation instance (see below)? 
https://github.com/loco-3d/crocoddyl/blob/master/bindings/python/crocoddyl/utils/vector-converter.hpp#L117

It is good to do not depend on internal choices that Pinocchio developers might make.
Without saying that a developer needs to go internally into Pinocchio for understanding how the binds are done.

Reply:
> Can we use our implementation instead (see below)?
https://github.com/loco-3d/crocoddyl/blob/master/bindings/python/crocoddyl/utils/vector-converter.hpp#L117
This is an aligned vector, and not an std::vector.

I'm not in favour of duplicating code in crocoddyl. I think the better place for keeping this class is in eigenpy rather than pinocchio, but @jcarpent can answer to that. However, I don't think duplicating upstream code at multiple places is good practice. It might have been okay to copy if we have to depend on a new library otherwise, but we already have this dependency.

Reply:
I agree with you @proyan, this should go into Eigenpy. Does @jcarpent  agree with us?

The question is what do we now (during this transition period)?
I see the following options:
  1. Use Crocoddyl implementation and wait for Eigenpy
  2. Use a mixture of implementations (Crocoddyl and Pinocchio); this is the current solution
  3. Remove the Crocoddyl option and update the entire code with Pinocchio (then substitute Pinocchio implementation once it has ported the code to Eigenpy)

I prefer to follow option 1 now. The reason is due to it requires less work for us.
I would discard the option 2 because it is a dirty solution.
I am OK if you want to commit for option 3 in this PR, but this is an avoidable effort from your side (I would say).

Reply:
I would suggest first applying 3. We will later for porting this code in EigenPy.

Reply:
The historical reason of this divergence is that
  1. Pinocchio doesn't have Pythonic support in the very begining.
  2. Pinocchio doesn't have std map support (not sure if it has now to be honest).

Reply:
It supports well std::map. Apparently, you are not following the current devs.

Reply:
I'll merge this PR for now. 
@cmastalli If you want, we can continue this conversation in a dedicated issue.

Reply:
No need to discuss.
I am happy if you handle in another PR.

Reply:
Just to be clear, I'm not in favor of duplicating code from one package to another. So any PR that I make would be to remove this duplication :)

Reply:
Thanks for ensuring we are in the same page.
Yes, I am in favor to remove the duplicated code as well.